File: sdkwork-app-sdk-python/sdkwork_app_sdk/models/music_create_form.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class MusicCreateForm:
    """创建音乐表单"""
    title: str = None
    description: str = None
    content_url: str
    format: str = None
    duration: int = None
    artist: str = None
    genre: str = None

File: sdkwork-app-sdk-typescript/src/types/music-create-form.ts
/** 创建音乐表单 */
export interface MusicCreateForm {
  /** 标题 */
  title?: string;
  /** 描述 */
  description?: string;
  /** 音乐URL */
  contentUrl: string;
  /** 格式 */
  format?: string;
  /** 时长(秒) */
  duration?: number;
  /** 艺术家 */
  artist?: string;
  /** 流派 */
  genre?: string;
}
